### DISCUSSION TITLE
Does this support M1 Machine Learningcores?

### DISCUSSION THREAD

**Discussion by bschollnick** [exadel-inc/CompreFace#406] [Ideas]
Does this support the Apple Silicon M1's Machine Learning Cores?

**Comment by pospielov** [comment]
no, as I'm aware Apple M1 is arm based processor and now we don't support such processors at all.
We are looking in this direction, but I can't say any timing when will add this.
But even more, to support M1's Machine Learning Cores we need to use a custom Tensorflow build, so it will require a separate docker image. So it will take more time to add this build to our flow.
